### PR TITLE
Exclude maps regardless of their group 

### PR DESCRIPTION
**Pull request by crescentrose**
And a ton of other smaller optimizations and cleanups.

### CODE REVIEW

**Comment by crescentrose** [comment]
Oops, didn't mean to do this here. My bad.

File: README.md [hunk 1]
```diff
@@ -1,114 +1,20 @@
-# NOTICE: UMC is Unsupported
-Hello everyone,
+# Uncletopia Map Chooser
 
-TL;DR: I will no longer be supporting UMC as a project and I will not be responding to issues/pull requests either. As it stands, it works as long as you are not trying to be too complex with the plugins or have large map lists of 100+ maps, but some features do not work as well as they could and it does not completely work with Workshop maps in its current form. If you wish to take on the project, please go ahead and fork the repo and do whatever you wish with my permission/blessing/whatever (you do not need to contact me!). Good luck! 
+This is a fork of [Universal Map Chooser](https://github.com/Silenci0/UMC) with
```
Comment (MAGNAT2645) [inline README.md:3]: It's `Ultimate Map Chooser`, not the `Universal`.
